### DISCUSSION TITLE
Table 9, Panel 2

### DISCUSSION THREAD

**Discussion by mbrazil1** [cssi-pi-community/2022-meeting#58] [Panel 2 roundtable (Monday afternoon)]
Topics discussed:

- Push for open source data and making data accessible. Is there sensitivity with storing data to the cloud? Do funding agencies require it to be stored in the US vs international and are there concerns about this?
- Data sovereignty 
- Data management plans 
- NIST, setting data standards and requirements, etc.
- Software, code - it is not enough to just make it open source - also needs to be accessible
- Is there follow-up (once an award is made), to ensure that data is captured? Should it be required in annual reports or somewhere else? 
-

**Comment by LSSchadler** [comment]
- Should there be a field, when reporting a publication where the data is stored?  
- Should NIST be funded and tasked with building the specs (at least in some fields - e.g. materials)
-
